src/prefect/core/task.py

```diff
@@ -125,11 +125,9 @@ def run(self, x, y):
         - cache_key (str, optional): if provided, a `cache_key` serves as a unique identifier for this Task's cache, and can
             be shared across both Tasks _and_ Flows; if not provided, the Task's _name_ will be used if running locally, or the
             Task's database ID if running in Cloud
-        - checkpoint (bool, optional, DEPRECATED): if this Task is successful, whether to
-            store its result using the `result_handler` available during the run; defaults to the value of
-            `tasks.defaults.checkpoint` in your user config.  Note that in the future, all tasks with result handlers
-            will be checkpointed.  Also note that checkpointing will only occur locally if `prefect.config.flows.checkpointing` is
-            set to `True`
+        - checkpoint (bool, optional): if this Task is successful, whether to
```

Comment:
Should the changelog reflect that this is no longer deprecated?

Reply:
great point, I'll update